jsonnet/versions.yaml

```diff
@@ -14,11 +14,11 @@ repos:
   thanos: openshift/thanos
 versions:
   alertmanager: 0.25.0
-  kubeRbacProxy: 0.14.0
+  kubeRbacProxy: 0.14.1
   kubeStateMetrics: 2.8.2
   nodeExporter: 1.5.0
   promLabelProxy: 0.6.0
   prometheus: 2.43.0
   prometheusAdapter: 0.10.0
   prometheusOperator: 0.63.0
-  thanos: 0.31.0
+  thanos: 0.30.2
```

Comment:
Could you please help me understand why addressing https://issues.redhat.com//browse/MON-3173 require versions to be changed? 
If it isn't strictly needed, we shouldn't introduce changes to the product in a PR that addresses a development workflow tweak.

Reply:
I explained this in the separate commit where I make the change: the versions job was failing because these versions were not aligned with the ones that are already used by the product.

Reply:
While I agree a separate PR would be ideal, I'd be happy to merge this since its in its own commit and clearly states why it was changed.

Reply:
If https://github.com/openshift/cluster-monitoring-operator/pull/1924 gets merged before this one, I'd discard my commit.

Reply:
> While I agree a separate PR would be ideal, I'd be happy to merge this since its in its own commit and clearly states why it was changed.

I am not against merging this but .. I wanted to understand why the versions are being downgraded. You might have more context than I do since the commit doesn't provide much information to me. It seems evident from the code that the versions are being downgraded but the message could be clearer about the reason behind this change.

E.g. Is it because the synbot disabled? 

```
   Synchronize version of thanos as it was downgraded and the sync bot is disabled

    Synchronize version of kubeRbacProxy

```


> the versions job was failing because these versions were not aligned with the ones that are already used in the product

If it isn't too much to ask for could you please incorporate that the message in the commit to indicate this. E.g. 

```
Fix versions ci job

`versions` ci job is failing because of versions of thanos and kube-rbac-proxy 
are different to the ones that are already used in the product.  This patch fixes
that by setting the correct version of thanos and kube-rbac-proxy. 
```

Reply:
:+1: for better docs, I have something in the pipeline for rhobs/handbook.

i'd argue for merging this. We need a bug to move this anyway and #1924 would need yet another one. So lets combine them and skip the additional Jira interaction.

Reply:
I can help with rhobs/handbook, it falls within https://issues.redhat.com/browse/MON-3217

Reply:
+1 on merging this. The question wasn't intended to block the merge.